.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+This PR:
+
+- [ ] Adds a new feature ...
+- [ ] Covered by tests
+- [ ] Fixes #nnnn
```

Comment:
```suggestion
This PR

* [x]
* [ ]

Follows #.
Related to #.
Fixes #.
```

Reply:
- It seems more obvious what to do with the checkmarks with they have some labels. 
- And I like a dedicated "Covered by tests": saves you from reminding an author that they need to add tests.

What do you think?

Reply:
👍